src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenByAuthorizationCodeParameters.cs

```diff
@@ -10,6 +10,8 @@ internal class AcquireTokenByAuthorizationCodeParameters : IAcquireTokenParamete
     {
         public string AuthorizationCode { get; set; }
 
+        public string PkceCodeVerifier { get; set; }
```

Comment:
nit: just `codeVerifier` as that's what you have elswhere

Reply:
I'd go the other way around - add PKCE prefix elsewhere.